.github/workflows/ci.yml

```diff
@@ -15,43 +15,43 @@ jobs:
           - os: ubuntu-latest
             java: 8
             distribution: zulu
-            jobtype: 1
+            jobtype: sbt-1
           - os: ubuntu-latest
             java: 8
             distribution: temurin
-            jobtype: 2
+            jobtype: sbt-2
           - os: ubuntu-latest
             java: 21
             distribution: temurin
-            jobtype: 1
+            jobtype: sbt-1
           - os: ubuntu-latest
             java: 24-ea
             distribution: temurin
-            jobtype: 1
+            jobtype: sbt-1
           - os: windows-latest
             java: 11
             distribution: temurin
-            jobtype: 1
+            jobtype: sbt-1
     env:
       # define Java options for both official sbt and sbt-extras
       JAVA_OPTS: -Xms2048M -Xmx2048M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
       JVM_OPTS:  -Xms2048M -Xmx2048M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
     steps:
     - uses: actions/checkout@v4
     - name: Setup JDK
-      uses: actions/setup-java@v3
+      uses: actions/setup-java@v4
       with:
         distribution: "${{ matrix.distribution }}"
         java-version: "${{ matrix.java }}"
         cache: sbt
     - uses: sbt/setup-sbt@v1
     - name: Build and test (sbt 1.x)
-      if: ${{ matrix.jobtype == 1 }}
+      if: ${{ matrix.jobtype == sbt-1 }}
```

Comment:
This failed - https://github.com/sbt/sbt-assembly/actions/runs/11425010983

```
The workflow is not valid. .github/workflows/ci.yml (Line: 49, Col: 11): Unrecognized named-value: 'sbt-1'. Located at position 19 within expression: matrix.jobtype == sbt-1 .github/workflows/ci.yml (Line: 54, Col: 11): Unrecognized named-value: 'sbt-2'. Located at position 19 within expression: matrix.jobtype == sbt-2
```

Reply:
I fixed the missing quotes